products/views.py

```diff
@@ -1,3 +1,33 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http           import JsonResponse
+from django.views          import View
+
+from products.models  import Product, ProductImage, TasteByProduct
+
+
+class MainProductView(View): 
+    def get(self, request): 
+        premiums             = Product.objects.all().order_by('-price')[:3]
+        fresh_products       = Product.objects.all().order_by('-roasting_date')[:4]
+        result_premium       = [{
```

Comment:
```suggestion
        fresh_products       = Product.objects.all().order_by('-roasting_date')[:4]
        
        result_premium       = [{
```
가독성을 위해 빈줄을 넣어주면 좋을 것 같습니다ㅎㅎ